### PR TITLE
feat(specs): add Conduit category properties

### PR DESCRIPTION
**Pull request by OlehChyhyryn**
asana: [[OpenEpd] Add conduit properties](https://app.asana.com/0/1207501793930589/1207868902793418/f)

### CODE REVIEW

File: src/openepd/model/specs/generated/electrical.py [hunk 1]
```diff
@@ -293,7 +293,26 @@ class LightingV1(BaseOpenEpdHierarchicalSpec):
...
+        example="100 mm",
+    )
+    outer_diameter: LengthMmStr | None = pyd.Field(
+        default=None,
+        description="The measurement of the distance of a straight line from one point on the outer wall of the pipe.",
+        example="100 mm",
+    )
+    inner_diameter: LengthMmStr | None = pyd.Field(
+        default=None,
+        description="The measurement of the distance of a straight line from one point on the inner wall of the pipe.",
```
Comment (alexche-cchange) [inline src/openepd/model/specs/generated/electrical.py:310]: This sentence looks incomplete - from one point on the inner wall to where? to the opposite wall? Or it is as is ?

Comment (alexche-cchange) [reply src/openepd/model/specs/generated/electrical.py:310]: Also, let's be consistent with the full stops at the end of description.

File: src/openepd/model/specs/generated/network_infrastructure.py [hunk 2]
```diff
@@ -169,7 +170,26 @@ class NetworkingRacewaysV1(BaseOpenEpdHierarchicalSpec):
...
+        default=None,
+        description="The measurement of the distance of a straight line from one point on the outer wall of the pipe.",
+        example="100 mm",
+    )
+    inner_diameter: LengthMmStr | None = pyd.Field(
+        default=None,
+        description="The measurement of the distance of a straight line from one point on the inner wall of the pipe.",
+        example="100 mm",
+    )
+    wall_thickness: LengthMmStr | None = pyd.Field(default=None, description="Conduit wall thickness", example="100 mm")
```
Comment (alexche-cchange) [inline src/openepd/model/specs/generated/network_infrastructure.py:190]: Maybe extract to a mixin? This seems to be super-close to the above.

**Review by alexche-cchange — APPROVED** [review]
see comments

File: src/openepd/model/specs/generated/electrical.py (hunk 1 above)
Comment (OlehChyhyryn) [reply src/openepd/model/specs/generated/electrical.py:310]: > This sentence looks incomplete - from one point on the inner wall to where? to the opposite wall? Or it is as is ?

to other point of inner wall. 

See scheme:

image

File: src/openepd/model/specs/generated/network_infrastructure.py (hunk 2 above)
Comment (OlehChyhyryn) [reply src/openepd/model/specs/generated/network_infrastructure.py:190]: Agree, will extract those properties to mixin

**Review by bohdanzahoruikocchangelabs — APPROVED** [review]

**Review by alexlitvinenko — APPROVED** [review]

File: src/openepd/model/specs/generated/electrical.py (hunk 1 above)
Comment (OlehChyhyryn) [reply src/openepd/model/specs/generated/electrical.py]: Slightly change the phrase

File: src/openepd/model/specs/generated/network_infrastructure.py (hunk 2 above)
Comment (OlehChyhyryn) [reply src/openepd/model/specs/generated/network_infrastructure.py]: Done

**Pull request merged**